lib/elixir_google_scraper_web/templates/keyword/_url_result_report.html.eex

```diff
@@ -0,0 +1,16 @@
+<div class="search-result-url">
+  <div class="search-result-url__panel-container">
+    <div class="search-result-url__panel">
+      <h4 class="search-result-url__title">Search Results Urls</h4>
+      <%= for url <- @keyword.search_result.result_urls do %>
+        <%= link url, class: "search-result-url__item d-inline-block text-truncate", target: "_blank", to: url %>
```

Comment:
Maybe we could move `d-inline-block text-truncate` into the `_search-result-url.scss` style?